assets/sass/patterns/molecules/social-links.scss

```diff
@@ -73,9 +72,14 @@
 
   }
 
+  .social-links__list_item {
+    @include padding(0 18 0 0);
+  }
+}
+
+@media only screen and (min-width: #{get-rem-from-px(900)}rem) {
```

Comment:
Can we not use magic numbers and use the variables ```$bkpt-site--x-wide```

Reply:
Done. The variables are now used instead of magic numbers.